docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -53,50 +64,53 @@ After the shebang and UTF-8 coding, add a `copyright line <https://www.linuxfoun
+Before committing your module documentation, please test it at the :ref:`command line and as HTML <dev_testing_module_documentation>`.
 
-Module documentation should briefly and accurately define what each module and option does, and how it works with others in the underlying system. Documentation should be written for broad audience--readable both by experts and non-experts.
-    * Descriptions should always start with a capital letter and end with a full stop. Consistency always helps.
-    * Verify that arguments in doc and module spec dict are identical.
-    * For password / secret arguments ``no_log=True`` should be set.
-    * For arguments that seem to contain sensitive information but **do not** contain secrets, such as "password_length", set ``no_log=False`` to disable the warning message.
-    * If an option is only sometimes required, describe the conditions. For example, "Required when I(state=present)."
-    * If your module allows ``check_mode``, reflect this fact in the documentation.
+After the shebang, the UTF-8 encoding, the copyright line, and the license section comes the ``DOCUMENTATION`` block. Ansible's online module documentation is generated from the ``DOCUMENTATION`` blocks in the source code of each module.
 
-To create clear, concise, consistent, and useful documentation, follow the :ref:`style guide <style_guide>`.
+The ``DOCUMENTATION`` block must be valid YAML. To make it easier:
 
-Each documentation field is described below. Before committing your module documentation, please test it at the command line and as HTML:
+* Start by copying our `example documentation string <https://github.com/ansible/ansible-documentation/blob/devel/examples/DOCUMENTATION.yml>`_.
+* Write the block in an :ref:`editor with YAML syntax highlighting <other_tools_and_programs>` before you include it in your Python file.
+* If you run into syntax issues that are difficult to resolve, use the `YAML Lint <http://www.yamllint.com/>`_ website to help validate the YAML.
 
-* As long as your module file is :ref:`available locally <local_modules>`, you can use ``ansible-doc -t module my_module_name`` to view your module documentation at the command line. Any parsing errors will be obvious - you can view details by adding ``-vvv`` to the command.
-* You should also :ref:`test the HTML output <testing_module_documentation>` of your module documentation.
+When writing module documentation, take the following statements into consideration:
 
+* Module documentation should briefly and accurately define what each module and option does and how it works with others in the underlying system.
+* Module documentation should be written for a broad audience and be easily understood both by experts and non-experts.
+* Descriptions should always start with a capital letter and end with a full stop or period. Consistency always helps.
+* Verify that arguments in doc and module spec dict are identical.
```

Comment:
is this still required? I thought this was validated magically now ( aka docs must match mod spec or the PR fails)?

Reply:
Let's try to remove it:) thanks